HomeWork3/ArrayTasksApp/Program.cs

```diff
@@ -0,0 +1,443 @@
+                    break;
+                case 4:
+                    MethodNamber_4();
+                    break;
+                case 5:
+                    MethodNamber_5();
+                    break;
+                case 6:
+                    MethodNamber_6();
+                    break;
+                case 7:
+                    MethodNamber_7();
+                    break;
+                case 8:
+                    MethodNamber_8();
+                    break;
+                case 9:
+                    MethodNamber_9();
+                    break;
+                case 10:
+                    MethodNamber_10();
+                    break;
+                default:
+                    throw new Exception("There is no such task !");
+            }
+        }
+
+        static void Description(int numberTask)//choice of description
+        {
+            Console.ForegroundColor = ConsoleColor.Green;
+
+            switch (numberTask)
+            {
+                case 1:
+                    Console.WriteLine("Write a program in C# Sharp to store elements in an array and print it. \n\n");
+                    break;
+                case 2:
+                    Console.WriteLine("Write a program in C# Sharp to read n number of values in an array and display it in reverse order.\n\n");
+                    break;
+                case 3:
+                    Console.WriteLine("Write a program in C# Sharp to find the sum of all elements of the array.\n\n");
+                    break;
+                case 4:
+                    Console.WriteLine("Write a program in C# Sharp to copy the elements one array into another array.\n\n");
+                    break;
+                case 5:
+                    Console.WriteLine("Write a program in C# Sharp to count a total number of duplicate elements in an array.\n\n");
+                    break;
+                case 6:
+                    Console.WriteLine("Write a program in C# Sharp to print all unique elements in an array.\n\n");
+                    break;
+                case 7:
+                    Console.WriteLine("Write a program in C# Sharp to merge two arrays of same size sorted in ascending order.\n\n");
+                    break;
+                case 8:
+                    Console.WriteLine("Write a program in C# Sharp to count the frequency of each element of an array.\n\n");
+                    break;
+                case 9:
+                    Console.WriteLine("Write a program in C# Sharp to find maximum and minimum element in an array.\n\n");
+                    break;
+                case 10:
+                    Console.WriteLine("Write a programin C# Sharp to separate odd and even integers in separate arrays.\n\n");
+                    break;
+                default:
+                    throw new Exception("There is no such task !");
+            }
+
+            Console.ResetColor();
+        }
+
+        static void MethodNamber_1()
```

Comment:
try not to use underscore in method names, just camelCase
```suggestion
        static void MethodNumber1()
```